### DISCUSSION TITLE
Unable to write square brackets in Jupiter lab 4.1

### DISCUSSION THREAD

**Discussion by simzap91** [jupyterlab/jupyterlab#15785] [Q&A]
I write square brackets with Alt+8/9 (Swedish keyboard, Macbook Pro 2016), and have been able to do so in Jupiter lab as well until the last update. Still works in the terminal but not in the notebook/code cells.

In the shortcuts settings I found that the command "application:toggle-side-tabbar" by default is set to Alt 1-9, I tried to edit that but it didn't solve the problem.

Any ideas?

Ps. New in programming so please use an easy to understand language.

thanks!

**Comment by krassowski** [comment]
@simzap91 please see https://github.com/jupyterlab/jupyterlab/issues/15744; this is a bug in 4.1.0 which will be fixed in 4.1.1 which we plan to release on Monday 12 Feb. In the meantime you can use a workaround from https://github.com/jupyterlab/jupyterlab/issues/15744#issuecomment-1936372773 or downgrade to 4.0.x. In the future please search issues before opening a new discussion/issue and please consider testing pre-releases to help catch issues like these early.

**Reply by simzap91** [reply]
Thank you, and noted!

**Reply by CarloNicolini** [reply]
It's still a problem in jupyterlab==4.1.5 on Mac, at least.

**Reply by krassowski** [reply]
Yes, https://github.com/jupyterlab/jupyterlab/issues/15744 was reopened because we had to revert the fix. The fix is coming back in https://github.com/jupyterlab/jupyterlab/pull/15927, hopefully in 4.2.0.
